include/grpc/grpc.h

```diff
@@ -351,6 +353,25 @@ typedef struct grpc_op {
   } data;
 } grpc_op;
 
+/* Propagation bits: this can be bitwise or-ed to form propagation_mask for
+ * grpc_call */
+/** Propagate deadline */
+#define GRPC_PROPAGATE_DEADLINE ((gpr_uint32)1)
+/** Propagate census context */
+#define GRPC_PROPAGATE_STATS_CONTEXT ((gpr_uint32)2)
```

Comment:
Since we're using the census name everywhere else, does it make sense to rename these similar to GRPC_PROPAGATE_CENSUS_STATS_CONTEXT? (I don't mind either way myself, just think of future readers of the code)